test/docker-compose.yml

```
@@ -0,0 +1,115 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version: '2.1'
+
+services:
+  oap:
+    extends:
+      file: base-compose.yml
+      service: oap
+    environment:
+      SW_STORAGE: h2
+
+  ui:
+    extends:
+      file: base-compose.yml
+      service: ui
+    ports:
+      - 8080
+    depends_on:
+      oap:
+        condition: service_healthy
+
+  provider:
+    build:
+      context: ./
+      dockerfile: docker/Dockerfile.provider
+    command: [ 'python3', '/entrypoint.py' ]
+    networks:
+      - e2e
+    expose:
+      - 9091
+    environment:
+      SW_AGENT_COLLECTOR_BACKEND_SERVICES: opa:11800
```

this address is wrong, how can the tests pass?

```suggestion
      SW_AGENT_COLLECTOR_BACKEND_SERVICES: oap:11800
```

---

There are setting in the [code](https://github.com/apache/skywalking-client-js/blob/728baf81793318fd63031cd5ff8dafd5541eb7fd/test/docker/provider.py#L23).

---

Then I think it is better to remove this from system env?